fitbenchmarking/fitbenchmarking
# Fixes #685: Adds vendor prefixes to support older versions of chrome + firefox

#### Description of Work

Fixes #685
Adds vendor prefixes for the max-width css property which were used before they were added to the common set of properties used in the modern versions of firefox, chrome, and safari.

Supported versions should now include all green sections on (this page)[https://caniuse.com/mdn-css_properties_max-width_max-content].


#### Testing Instructions

1. Run fitbechmarking and check the outputs look as expected on a variety of browesrs including:
   - Firefox <= 65.0.2
   - Firefox >= 66.0.0
   - Chrome
   - Seamonkey on wsl?

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
